tests/cupy_tests/manipulation_tests/test_rearrange.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 
+import numpy
```

Comment:
Please add a blank line after `import numpy`.
We group imports in three categories: standard libraries, third-party libraries, internal imports.
(https://docs.chainer.org/en/v6.0.0b1/contribution.html#coding-guidelines)

Reply:
done